### DISCUSSION TITLE
Can we pass a callback to handleSubmit()?

### DISCUSSION THREAD

**Discussion by Nilesh9768** [data-driven-forms/react-forms#1332] [Need help]
Hi Team,
I have a use case where I want to call a function after handleSubmit is completely executed.
I tried `handleSubmit(callbackFunction)`, but it doesn't work, even `await handleSubmit()` doesn't work.
How can we achieve this?

Thank you!

**Comment by rvsia** [comment]
Hello,

`handleSubmit` (= `onSubmit` from the renderer) is completely your own function, so you do not need to use any callbacks.
